app/views/idv/otp_delivery_method/new.html.erb

```diff
@@ -49,13 +49,14 @@
       </div>
     </label>
 
-    <% if FeatureManagement.enable_gpo_verification? %>
+    <% if gpo_letter_available %>
```

Comment:
There's a minor visual difference here now, where currently there's 2rem gap around the options. When this is `false`, that'll now be reduced to 1rem on the bottom of the options.

I think ideally the margin would be set on some wrapper of the two options, but it's a bit complicated with the current markup and margin collapsing behavior.

A more direct option to preserve the current behavior could be to always render an element with the margin, and take advantage of the margin collapsing.

```suggestion
    <div class="margin-top-4"></div>
    <% if gpo_letter_available %>
```

Reply:
Ah, good catch, thank you!